docs/release-notes/8.3.asciidoc

```diff
@@ -0,0 +1,81 @@
+== 8.3.0
+
+[discrete]
+[[breaking-changes-8.3.0]]
+==== Breaking changes
+// tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+:pull: https://github.com/elastic/kibana/pull/
+* Adds confirmation modal with docs link when updating rules if V1/V2 jobs are installed ({pull}128334[#128334]).
+// end::breaking-changes[]
+
+[discrete]
+[[features-8.3.0]]
+==== Features
+* Adds new fields to the rule model: Related Integrations, Required Fields, and Setup {pull}132409[#132409]
+* 131827 Update Detections Response view with pagination and opening numbers in timeline {pull}131828[#131828]
+* Adds UI for new Rule Fields: Related Integrations, Required Fields, and Setup {pull}131475[#131475]
+* 131028 implement recently opened cases table {pull}131029[#131029]
+* Adds Dashboards and Threat Hunting Landing pages {pull}130905[#130905]
+* Cases by status {pull}130670[#130670]
+* Adds risk tab to the user details page {pull}130256[#130256]
+* Adds the authentication tab to the user details page {pull}129456[#129456]
+* Open alerts by Rule {pull}129021[#129021]
+* Adds to timeline button {pull}128596[#128596]
+* Hosts and Users by severity {pull}128335[#128335]
+* Soc dashboard - UI {pull}128087[#128087]
+* Allows multiple alerts to be added to a case ({pull}130958[#130958]).
+* Adds the option to delete case comments multiple alerts to be added to a case ({pull}130254[#130254]).
+* Provides an option to select a severity level for a case ({pull}131626[#131626]).
+* Adds the *Alerts* tab to cases ({pull}131883[#131883]).
+* Adds the *Average time to close* metric to the Cases page ({pull}131909[#131909]).
+
+[discrete]
+[[bug-fixes-8.3.0]]
+==== Bug fixes and enhancements
+* Separates array values in the Alerts table with commas ({pull}133297[#133297]).
+* Exposes the EQL search configurations `event_category_field`, `tiebreaker_field`, and `timestamp_field` through the rules API and UI for Event Correlation rules ({pull}132247[#132247]).
+* Adds the *Session ID* field to the *Highlighted fields* section on the Alert details flyout ({pull}132219[#132219]).
+* Introduces a new design for the main navigation menu ({pull}132210[#132210], {pull}131437[#131437]).
+* Adds Dashboards and Threat Hunting Landing pages ({pull}130905[#130905]).
+* Introduces the Detections & Response dashboard ({pull}130670[#130670], {pull}129021[#129021], {pull}131029[#131029], {pull}128335[#128335], {pull}128087[#128087]).
+** Update Detections Response view with pagination and opening numbers in Timeline ({pull}131828[#131828], {pull}131029[#131029]).
+* Allows highlighted fields to be investigated in Timeline ({pull}131255[#131255]).
+* Adds support to Osquery for multi-line queries ({pull}131224[#131224]).
+* Improves the performance of these actions on the bulk rule actions endpoint ({pull}130924[#130924]):
```

Comment:
That looks good enough to me, thanks @nastasha-solomon 👍 

If I was nitpicking, I'd either formulate it around the bulk actions menu in the UI (so not mentioning the endpoint) or around the endpoint parameters.

If we'd emphasize the UI, it could be:

```
* Improves the performance of these bulk actions on rules ({pull}130924[#130924]):
** Add index patterns
** Delete index patterns
** Add tags
** Delete tags
** Apply Timeline template
```

If we'd emphasize the endpoint, it could be:

```
* Improves the performance of editing actions (`"action": "edit"`) supported by the `/api/detection_engine/rules/_bulk_action` endpoint ({pull}130924[#130924]). This covers the following actions available in the Rules table UI:
** Add index patterns
** Delete index patterns
** Add tags
** Delete tags
** Apply Timeline template
```

Note that we don't have "Set tags" and "Set index patterns" exposed as menu items in the UI.

Reply:
Thanks, @banderror ! I noticed the difference between the UI and the endpoint as well and modeled the list against what was listed for the endpoint in [these docs](https://www.elastic.co/guide/en/security/master/bulk-actions-rules-api.html#bulk-edit-object-schema) because it looked closer to what was mentioned in the PR description of https://github.com/elastic/kibana/pull/130924. Do you have a preference on which we focus on for the release notes?

Reply:
Ahh doc bug emerges! 

![doc-bug-emerges](https://user-images.githubusercontent.com/79124755/175073691-7390de18-25c3-4226-a043-b4add36075ab.gif)

Would you mind filing a doc bug for this in the `security-docs` repo @banderror so we have all the necessary deets?

Reply:
@nastasha-solomon will do. Or will just open a PR for that. It should be a tiny change.

Reply:
You opening a PR will probably be faster right now. Feel free to tag me and any of the other doc team members on the PR as reviewers - thanks!

Reply:
https://github.com/elastic/security-docs/pull/2130